docs/source/app-dev/authorization.rst

```diff
@@ -157,14 +157,42 @@ the participant user management service *without* issuing new access tokens,
 as would be required for the custom Daml claims tokens.
 
 User access tokens are `JWTs <https://datatracker.ietf.org/doc/html/rfc7519>`_ that follow the
-`OAuth 2.0 standard <https://datatracker.ietf.org/doc/html/rfc6749>`_ with a JSON payload of the following format.
+`OAuth 2.0 standard <https://datatracker.ietf.org/doc/html/rfc6749>`_. There are two
+different JSON encodings: An audience-based token format that relies
+on the audience field to designate that it is designed for a specific
+Daml participant and a scope-based audience format which relies on the
+scope field to designate. Both formats can be used interchangeably but
+if possible, use of the audience-based token format is recommend as it
+is compatible with a wider range of IAMs, e.g., Kubernetes does not
```

Comment:
Is it really Kubernetes problem or more like service mesh products like Istio, traefik, etc?

Reply:
It is k8s, k8s can issue tokens itself but doesn’t let you set `scope`.